examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala

```diff
@@ -174,6 +174,7 @@ object LDAExample {
 
     // Get dataset of document texts
     // One document per line in each text file.
+    // One partition each text file. Consider using coalesce as necessary.
```

Comment:
As long as we're documenting this, let's edit this a bit more. It's not guaranteed that there will be a partition per text file. I'd say something more like:

"If the input consists of many small files, this can result in a large number of small partitions, which can degrade performance. In this case, consider using coalesce() to create fewer, larger partitions."